### ISSUE TITLE
The create-symlinks.sh script does not remove existing links in extensions folder 

### ISSUE THREAD

**Issue by pastakhov** [CanastaWiki/Canasta#321]
How to reproduce: restart a web docker container without recreating one. You will see the error messages like:
```
Symlinking bundled extensions...
ln: failed to create symbolic link '/var/www/mediawiki/w/extensions/AJAXPoll/AJAXPoll': File exists
...
```
For each extension.

Most likely, it is the reason for #300 

When you remove an extension from a folder mounted to the `user-extensions` folder and restart the container you expect that the symlink in the extension folder will be changed to use the Canasta bundled extension. However because the script does not delete existing symbolic links, the symlink to the removed extension from the `user-extensions` folder will not be replaced.

**Comment by pastakhov** [comment]
I agree, the script is not working, but I didn't test/check the monitor-directories.sh script. I thought it was removed or disabled. It can be related to this issue (or be similar), but it is another one. This issue describes a mistake in the create-symlinks.sh script.  In other words, the monitor-directories.sh does not affect this issue but can have similar mistake in the code (and yes, it does not work).

**Comment by yaronkoren** [comment]
That's true - the problem needs to be fixed in create-symlinks.sh. I still think fixing monitor-directories.sh (or maybe just getting it to always run) will fix this problem 99% of the time, but there will still be cases when a "bad" symlink is there when Canasta is started, and for that, create-symlinks.sh needs to take action.

**Comment by yaronkoren** [comment]
I believe this was fixed by PR #366 ... I'm closing this now.
